speakercard.js

```diff
@@ -0,0 +1,105 @@
+  {
+    name: 'Jane Doe',
+    title: 'CEO of Company',
+    image: './Assets/model2.jpg',
+    text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.',
+  },
+  {
+    name: 'John Doe',
+    title: 'CEO of Company',
+    image: './Assets/model3.jpg',
+    text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.',
+  },
+  {
+    name: 'Jane Doe',
+    title: 'CEO of Company',
+    image: './Assets/model4.jpg',
+    text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.',
+  },
+  {
+    name: 'John Doe',
+    title: 'CEO of Company',
+    image: './Assets/model5.jpg',
+    text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.',
+  },
+  {
+    name: 'Jane Doe',
+    title: 'CEO of Company',
+    image: './Assets/model6.jpg',
+    text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.',
+  },
+];
+
+// Create a function to create a card
+
+const speakerSection = document.querySelector('#jsSection');
+const button = document.querySelector('.moreButton');
+const x = window.innerWidth;
+let num = 2;
+if (x > 768) {
+  num = 6;
+} else {
+  num = 2;
+}
+for (let i = 0; i < num; i += 1) {
+  speakerSection.innerHTML += `
+   <div class="speaker">        
+        <img src="${speakerArray[i].image}" alt="speaker1">
+        <div class="speakerInfo">
+            <h3>${speakerArray[i].name}</h3>
+            <p class="position">${speakerArray[i].title}</p>
+            <p>${speakerArray[i].text}</p>
+        </div>      
+        
+    </div>   
+   `;
+}
+
+button.addEventListener('click', () => {
+  num += 4;
+  if (num > 6) {
+    num = 6;
+  }
+  speakerSection.innerHTML = '';
+  for (let i = 0; i < num; i += 1) {
+    speakerSection.innerHTML += `
+        <div class="speaker">        
+             <img src="${speakerArray[i].image}" alt="speaker1">
+             <div class="speakerInfo">
+                 <h3>${speakerArray[i].name}</h3>
+                 <p class="position">${speakerArray[i].title}</p>
+                 <p>${speakerArray[i].text}</p>
+             </div>      
+             
+         </div>   
+        `;
+  }
+  if (num === 6) {
+    button.innerHTML = 'Less<i class="fas fa-chevron-up"></i>';
+    button.addEventListener('click', () => {
+      num = 2;
+      speakerSection.innerHTML = '';
+      for (let i = 0; i < num; i += 1) {
+        speakerSection.innerHTML += `
+                <div class="speaker">        
+                     <img src="${speakerArray[i].image}" alt="speaker1">
+                     <div class="speakerInfo">
+                         <h3>${speakerArray[i].name}</h3>
+                         <p class="position">${speakerArray[i].title}</p>
+                         <p>${speakerArray[i].text}</p>
+                     </div>      
+                     
+                 </div>   
+                `;
+      }
+      button.innerHTML = 'More<i class="fas fa-chevron-down"></i>';
```

Comment:
[OPTIONAL] I think you did a great job with the MORE button, however, kindly make sure that the button is still working after clicking on it twice. The button stops working when trying to click on it a third time. :eyes: :eyes: 
![image](https://user-images.githubusercontent.com/98363075/197112696-6d26347a-3b33-4ac2-be89-bc58b93feaf9.png)